src/Workspaces/Core/Portable/CodeActions/Annotations/NavigationAnnotation.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.CodeActions
+{
+    /// <summary>
+    /// Apply this annotation to an appropriate Syntax element to request that it should be 
```

Comment:
Can this more carefully spec whether this will navigate to the start or end of the token? Or if it's a node is something selected?